.travis.yml

```diff
@@ -2,4 +2,7 @@ language: java
 jdk:
   - oraclejdk8
 
-script: "mvn clean install"
+script: "mvn clean install && mvn cobertura:cobertura"
```

Comment:
Do we need to add cobertura plugin in the pom file or it's already existing?

Reply:
exist already!